tests/func-tests/node_placement_test.go

```diff
@@ -124,11 +206,11 @@ func updatePodAssignments(pods *v1.PodList, podMap map[string]bool, nodeType str
 	}
 }
 
-func listPodsInNode(client kubecli.KubevirtClient, nodeName string) *v1.PodList {
-	pods, err := client.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.TODO(), k8smetav1.ListOptions{
+func listPodsInNode(g Gomega, client kubecli.KubevirtClient, nodeName string) *v1.PodList {
+	pods, err := client.CoreV1().Pods("kubevirt-hyperconverged").List(context.TODO(), k8smetav1.ListOptions{
```

Comment:
This is a change I keep doing locally, because I haven't figured out how to pass a value to `flags.KubevirtInstallNamespace` during the test run through my IDE. Forgot to change it back to original value. 🤦🏾‍♂️